admin-ui/app/styles/main.scss

```
@@ -457,7 +457,7 @@ html {
 .month-picker .tab.btn:before {
     font-style: normal;
     font-weight: normal;
-    font-family: "icomoon";
+    font-family: "icomoon", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "icomoon" (font-family-name-quotes)

(font-family-name-quotes)